app/services/proxy_api/external_ipam.rb

```diff
@@ -226,9 +226,9 @@ def get_subnet(subnet, group = "")
     def ip_exists(ip, subnet, group = "")
       raise "subnet cannot be nil" if subnet.nil?
       raise "ip cannot be nil" if ip.nil?
-      response = parse get("/subnet/#{subnet}/#{ip}", query: {group: group })
+      response = parse get("/subnet/#{subnet}/#{ip}", query: { group: group })
       raise(response['error']) if response.is_a?(Hash) && response['error'].present?
-      response
+      response['result']
```

Comment:
Should this still be able to deal with the old Proxy version? In general we try to support first upgrading Foreman and then at a later time all the proxies. In this case you can use `is_a?(Hash)`, probably combining the code with the above `raise()`. Something like
```ruby
if response.is_a?(Hash)
  raise(response['error']) if response['error'].present?

  response['result']
else # Foreman Proxy < 3.3 support
  response
end
```

Reply:
Good point - This proposed change will be able to handle old proxy versions